docker-entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+until nc -z -v -w30 "$DATABASE_HOST" 5432
```

Comment:
is it the right port for the DB container? it is exposed on `5433` 🤔

Reply:
That is the port that it is exposed to the host with yes, but itself, inside the network, exposes on the 5432 port. So for other containers, `profile-db` will be accessible from `profile-db:5432` and from the host machine, it will be accessible from `localhost:5433`.

Reply:
ok, gotcha! 👍